### PR TITLE
Add default value to the @param tag for the constructor in /wp-admin/includes/class-custom-image-header.php

### PR DESCRIPTION
**Pull request by faisalahammad**
I added "Default false" to the @param tag for the $admin_image_div_callback parameter, so that it is clear to developers that this parameter is optional and its default value is false.

Trac ticket: [58049](https://core.trac.wordpress.org/ticket/58049)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/class-custom-image-header.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ class Custom_Image_Header {
 	 * Constructor - Register administration header callback.
 	 *
 	 * @since 2.1.0
-	 * @param callable $admin_header_callback
+	 * @param callable $admin_header_callback. Default false.
```
Comment (cadic) [inline src/wp-admin/includes/class-custom-image-header.php]: In the PR description and Trac Ticket you mentioned `$admin_image_div_callback` param to be updated, but the actual change is related to the `$admin_header_callback`

```suggestion
	 * @param callable $admin_header_callback.
```

File: src/wp-admin/includes/class-custom-image-header.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ class Custom_Image_Header {
...
 	 * @param callable $admin_image_div_callback Optional custom image div output callback.
```
Comment (cadic) [inline src/wp-admin/includes/class-custom-image-header.php]:
```suggestion
	 * @param callable $admin_image_div_callback Optional custom image div output callback. Default empty string.
```

File: src/wp-admin/network/user-new.php [hunk 3]
```diff
@@ -100,7 +100,8 @@
 $title       = __( 'Add New User' );
 $parent_file = 'users.php';
 
-require_once ABSPATH . 'wp-admin/admin-header.php'; ?>
```
Comment (cadic) [inline src/wp-admin/network/user-new.php]: This doesn't look like the related change to the current Trac Ticket.

**Review by cadic — CHANGES_REQUESTED** [review]
@faisalahammad thank you for the PR, looks like the right parameter to update the doc is `$admin_image_div_callback`, could you please check.

File: src/wp-admin/includes/class-custom-image-header.php (hunk 1 above)
Comment (faisalahammad) [reply src/wp-admin/includes/class-custom-image-header.php]: Thank you @cadic 
I've made changes and sent another PR. 

Could you please check and let me know how that goes?

File: src/wp-admin/network/user-new.php (hunk 3 above)
Comment (faisalahammad) [reply src/wp-admin/network/user-new.php]: Thank you, I've reverted the changes.

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57169](https://core.trac.wordpress.org/changeset/57169).

**Pull request closed without merging**